src/sentry/event_manager.py

```diff
@@ -2836,6 +2841,25 @@ def _detect_performance_problems(
         )
 
 
+def _update_user_reports_with_event_link(job: Job, group_info: GroupInfo) -> None:
+    metrics.incr("event_manager.save._update_user_reports_with_event_link")
+    event = job["event"]
+    project = event.project
+    user_reports_without_group = UserReport.objects.filter(
```

Comment:
Maybe group_id_is_null OR environment_id_is_null? Not sure if this case happens

Reply:
does not happen.